### DISCUSSION TITLE
Updates?

### DISCUSSION THREAD

**Discussion by kentcdodds** [johnlindquist/kit#152] [General]
Quick question: Is the script kit app automatically updated? I just noticed the UI changed slightly and didn't manually update it or anything.

If not, what's the best way to update it?

**Comment by johnlindquist** [comment]
Yeah, Script Kit checks for updates on the system "resume" event AKA "wake from sleep". Then it should post a notification that it's relaunching.

"Check for updates" is under the "Help" tab.
